src/entry-server.js

```diff
@@ -3,7 +3,7 @@
 
 import { isFunction } from 'lodash-es';
 
-export default function initializeServer(createApp, serverOpts) {
+export default function initializeServer(createApp, serverOpts, runtimeConfig) {
```

Comment:
Should we ever have default values for `serverOpts` or `runtimeConfig`? Something like `runtimeConfig = {}`?

Reply:
Let's just use a new `serverOpts.additionalContext` field, and it can be set to default to `null` below (line 10-11)

Reply:
That makes it a little more generic in that we can pass additional stuff needed by the `index.tpl.html` render step, and keeps the exposed API the same